webapp/src/main/resources/public/js/ict/IctModal.js

```diff
@@ -64,6 +62,18 @@ class IctModal extends React.Component {
                         )}           
                     </Modal.Body>
                     <Modal.Footer>
+                        {this.props.experimentButtons['global'] !== undefined &&
```

Comment:
you can introduce a function here to "render" an action button. The code is duplicated here (global and locale)

And then to keep things generic you should iterates on the button list and render each button. Potentially there could be only 1 or maybe more than 3. We shouldn't hardcode the logic with 2 buttons